### PR TITLE
Fix/update document

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -99,6 +99,23 @@ To be able to execute JUnit5 Tests with IntellIJ, you need at *least IntellIJ 20
 ==== Use the Selenium Grid
 
 [source,shell]
+
+
+[NOTE]
+--
+To run tests on *PLF 5.2.x* and *PLF 5.1.x* you must ignore *Answer* TC by adding :
+
+ `-Dselenide.test.tags.exclude=answer \`
```
Comment (thomasdelhomenie) [inline README.adoc]: Isn't there a way to disable some tests by default ?

File: README.adoc [hunk 2]
```diff
@@ -87,7 +87,7 @@ If you want to execute the *tests with Selenium Grid, you can skip this step*.
 
 [IMPORTANT]
 --
-To be able to execute JUnit5 Tests with IntellIJ, you need at *least IntellIJ 2016.3*
+To be able to execute JUnit5 Tests with IntellIJ, you need at *least IntellIJ 2018.x*
```
[the remark this answers is not in the document]
Comment (ilyeskallala) [reply README.adoc]: yes, it does not work anymore with old versions since we upgrade junit to 5.0.0

File: README.adoc (hunk 1 above)
Comment (ilyeskallala) [reply README.adoc]: yes I added this properties to pom.xml 
<selenide.test.tags.exclude>answer</selenide.test.tags.exclude>

File: README.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (thomasdelhomenie) [reply README.adoc]: ok

File: README.adoc (hunk 1 above)
Comment (thomasdelhomenie) [reply README.adoc]: ok.
Is it necessary to say "*Answer* TC are disabled" ?

Comment (ilyeskallala) [reply README.adoc]: no, just to inform that answer tests will not be executed.

Comment (thomasdelhomenie) [reply README.adoc]: so you prefer to keep it or remove it ?

Comment (ilyeskallala) [reply README.adoc]: I removed it.

**Pull request merged**